.github/workflows/release-packages.yml

```diff
@@ -0,0 +1,71 @@
+name: Build packages for a release
+
+on:
+  push:
+    tags:
+    - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build-linux:
+    name: Build artifacts
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+        - os: macos-latest
+          BUILD_OS_NAME: macos
```

Comment:
This was `osx` with Travis (eg. `core-geth-alltools-osx-v1.12.1.zip`). Is there a reason to use `macos` instead?

Reply:
(I generally think `macos` is a "better" name (since maybe one day in a distant galaxy Apple with finally make the major version bump... , and then OSX would be OSXI ;)
But just thinking about users' programmatic download concerns/dependencies.

Reply:
Nope, my bad. Nice catch man, multi cursor magic, while copying the name from the line above. Resolved by c4422acb